packages/coding-standards/src/CsFixer/ForbiddenDumpFixer.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\CodingStandards\CsFixer;
+
+use PhpCsFixer\Fixer\DefinedFixerInterface;
+use PhpCsFixer\Fixer\FixerInterface;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\FixerDefinition\FixerDefinitionInterface;
+use PhpCsFixer\Tokenizer\Tokens;
+use SplFileInfo;
+
+final class ForbiddenDumpFixer implements FixerInterface, DefinedFixerInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getDefinition(): FixerDefinitionInterface
+    {
+        return new FixerDefinition(
+            'Removes forgotten dumps from twig',
+            [
+                new CodeSample('{{ d(123) }}'),
+                new CodeSample('{{ dump(1234) }}'),
+            ]
+        );
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function isCandidate(Tokens $tokens): bool
+    {
+        return true;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function isRisky(): bool
+    {
+        return false;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function fix(SplFileInfo $file, Tokens $tokens): void
+    {
+        $code = preg_replace_callback(
+            '@[{]{2}\s*(d|dump)\s*[(]\s*.+?\s*[)]\s*[}]{2}@imsu',
```

Comment:
I found `@{{\s*(d|dump)\s*\(\s*.+?\s*\)\s*}}@imsu` more readable. What do You think?